Sources/DeclarationHelpers.swift

```diff
@@ -164,12 +164,17 @@ enum Declaration: Hashable {
 
     /// Whether or not this declaration represents a stored instance property
     var isStoredInstanceProperty: Bool {
-        guard keyword == "let" || keyword == "var" else { return false }
-
         // A static property is not an instance property
-        if modifiers.contains("static") {
-            return false
-        }
+        modifiers.contains("static") ? false : isStoredProperty
```

Comment:
Better to use boolean logic than a ternary operator

```suggestion
        !modifiers.contains("static") && isStoredProperty
```